pyfar/dsp/filter/frequency_weighting.py

```diff
@@ -37,11 +37,11 @@ def frequency_weighting_filter(
     .. note::
         This function will run a least-squares algorithm to iteratively
         approach the filter coefficients for the target weighting curve.
-        The coefficients are cached, but only when the ``error_weighting``
+        The coefficients are cached, but only when the `error_weighting`
         and ``kwargs`` arguments are unused, which are unhashable.
```

Comment:
Should also use single quotes

```suggestion
        and `kwargs` arguments are unused, which are unhashable.
```